saleor/core/utils/random_data.py

```diff
@@ -574,7 +574,7 @@ def create_fake_payment(mock_notify, order):
         total=order.total.gross.amount,
         currency=order.total.gross.currency,
     )
-    manager = get_plugins_manager()
+    manager = get_plugins_manager(allow_replica=False)
```

Comment:
Why are we disallowing replicas here? Of all the places that this PR touches, I think only the `PluginUpdate` mutation can affect the list of loaded plugins, so there we should disallow replica, but in other places, we should be able to assume that the replica is up-to-date and can be used.

Reply:
I assumed that whenever write to database we should have up-to-date results.